Finish rename to literm, including for settings

special opened the pull request:
Rename the last few things that reference 'fingerterm'. The only one
that actually matters is the settings path, which will break your
existing config, but you'll live with it.

rburchell commented:
Thanks. I haven't been using the executable much, perhaps I should do so.

plfiorini commented:
It's pretty convenient, qmlscene works too of course but for a system-wide install people are going to use the demo program instead

special committed:
Finish rename to literm, including for settings

Rename the last few things that reference 'fingerterm'. The only one
that actually matters is the settings path, which will break your
existing config, but you'll live with it.

rburchell commented:
I'd like to leave the config for now. That one I think will need migration handling (since there are a number of existing mobile users out there). Can you revert that part? Otherwise OK.

special committed:
Migrate FingerTerm settings to literm if present

I guess this would be less intrusive as a copy, but that is annoying.

special commented:
That commit will move FingerTerm settings to literm if present. Which is annoying if you still wanted your FingerTerm settings, I suppose, but I don't really want to bother writing directory copying.

If you don't like that one, I'll just revert those, but I dislike the FingerTerm config path.

rburchell commented:
Thanks, perfect!
